src/components/schema-tree.js

```diff
@@ -244,7 +244,12 @@ export default class SchemaTree extends LitElement {
                 : ''
           }
           <span class="${dataTypeCss}">
-            ${dataType === 'array' ? `${type}[]` : `${type}`}
+            ${dataType === 'array'
+            ? type.match(/integer|number/g)
+            ? html`[<span class="formnum">${format}</span>]`
+            : `[${type}]` : type.match(/integer|number/g)
+            ? html`${type}(<span class="formnum">${format}</span>)`
```

Comment:
There is already a number class, let's reuse that one instead `number` or `integer` in the `class`, not hard coded to a new class.

Reply:
i need different colour here to notice exact type of integer

Reply:
This is potentially going to be hard sell for others. Can we focus on fixing the display of the array and type in this PR and level coloring it differently to a later PR?